servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/MultiAddressUrlHttpClientTest.java

```diff
@@ -92,6 +93,7 @@ public static void beforeClass() throws Exception {
         afterClassCloseables = newCompositeCloseable();
 
         client = afterClassCloseables.append(HttpClients.forMultiAddressUrl()
+                .socketOption(CONNECT_TIMEOUT, 1) // windows default connect timeout is seconds, we want to fail fast.
```

Comment:
Could you please elaborate why this is necessary? Does it cause issues with the test if windows has different default value here?

Reply:
yes, tests which expect connection failures to trigger retries and ultimately failures on my windows installation were taking 2 seconds to fail each connection attempt. This often lead to test timeouts when we do 5 retries and have a 10 second test timeout.

Reply:
Thanks!